immuni_exposure_ingestion/apis/ingestion.py

```diff
@@ -89,6 +89,10 @@
 @validate(
     location=Location.JSON,
     province=Province(),
+    country_of_interest=fields.List(
+        fields.String(validate=Regexp(r"^[A-Z]{2}$")),
```

Comment:
We should extrapolate this `r"^[A-Z]{2}$"` and put it in a variable in common. I added a similar comment in `immuni-backend-exposure-reporting` before seeing this. I'd move all to common. For consistency, I'd create a dedicated model in common as it has been done for `Province`.

Reply:
Done, as suggested ! 🙏